File: sc-client/src/app/types/magazineType/magazine.ts
import {MembershipFeeType} from './membershipFeeType';
import {Editor} from './editor';
import {AreaOfScience} from './areaOfScience';
import {PaymentType} from './paymentType';



export interface Magazine {

   id: number;
   title: string;
   issn: number;
   membershipFeeType: MembershipFeeType;
   editors: Editor[];
   areasOfScience: AreaOfScience[];
  paymentTypes: PaymentType[];

}

File: scientific-center/src/main/java/com/scientific/center/magazine/Magazine.java
package com.scientific.center.magazine;

import java.util.List;

import lombok.Builder;
import lombok.Value;

@Value
@Builder
public class Magazine {

	private Long id;
	private String title;
	private Integer issn;
	private MembershipFeeType membershipFeeType;
	private List<Editor> editors;
	private List<AreaOfScience> areasOfScience;
	private List<PaymentType> paymentTypes;
}
